File: python/Elektronik.py
from Produk import Produk

class Elektronik(Produk):
    def __init__(self, id, Nama_produk, Harga, Stok, GambarPath, Kategori_produk, Deskripsi_produk):
        super().__init__(id, Nama_produk, Harga, Stok, GambarPath)
        self.Kategori_produk = Kategori_produk
        self.Deskripsi_produk = Deskripsi_produk

    def display(self):
        super().display()
        print(f"{self.Kategori_produk:<20} {self.Deskripsi_produk:<50}", end=" ")

File: c++/Elektronik.cpp
#include "Elektronik.h"
#include <iomanip>

Elektronik::Elektronik(int id, std::string Nama_produk, float Harga, int Stok, std::string GambarPath, std::string Kategori_produk, std::string Deskripsi_produk)
    : Produk(id, Nama_produk, Harga, Stok, GambarPath), Kategori_produk(Kategori_produk), Deskripsi_produk(Deskripsi_produk) {}

void Elektronik::display() const {
    Produk::display();
    std::cout << std::left
              << std::setw(20) << Kategori_produk
              << std::setw(50) << Deskripsi_produk;
}
